logback-contrib/mongodb/access/pom.xml

```diff
@@ -0,0 +1,44 @@
+<!--
```

Comment:
No no we are not interested in mongoDB let's drop this part logback-json contribs...